### PR TITLE
feat(sdk/ui): update Badge component

### PR DESCRIPTION
**Pull request by bmc08gt**
(Previous PR was auto closed with no way to retarget or reopen)

Screenshot 2024-01-23 at 4 16 56 PM

### CODE REVIEW

File: sdk/src/commonMain/kotlin/ai/botstacks/sdk/ui/views/Badge.kt [hunk 1]
```diff
@@ -12,29 +11,48 @@ import androidx.compose.ui.unit.dp
...
-        fontSize = 10.sp,
-        modifier = Modifier.align(Alignment.Center)
-      )
+fun Badge(
+    count: Int,
+    modifier: Modifier = Modifier
+) {
+    if (count > 0) {
+        val text = when {
+            count in 1..98 -> "$count"
```
Comment (thebrentley) [inline sdk/src/commonMain/kotlin/ai/botstacks/sdk/ui/views/Badge.kt:27]: Shouldn't this be 1..99?

**Review by thebrentley — CHANGES_REQUESTED** [review]

File: sdk/src/commonMain/kotlin/ai/botstacks/sdk/ui/views/Badge.kt (hunk 1 above)
Comment (bmc08gt) [reply sdk/src/commonMain/kotlin/ai/botstacks/sdk/ui/views/Badge.kt:27]: 99+ is handled in the else. 1..98 is 1 >= X >= 98

Comment (thebrentley) [reply sdk/src/commonMain/kotlin/ai/botstacks/sdk/ui/views/Badge.kt:27]: If you have 99 notifications, I'd expect it to say 99 rather than 99+. Right?

**Comment by bmc08gt** [comment]
@thebrentley PR's good for re-review

**Review by thebrentley — APPROVED** [review]

**Pull request merged**